features/net/network-socket/WiFiInterface.cpp

```diff
@@ -60,12 +60,3 @@ int WiFiInterface::set_credentials(const char *ssid, const char *pass, nsapi_sec
     return 0;
 }
 
-int WiFiInterface::connect()
```

Comment:
If this is removed, how does the WiFiInterface override NetworkInterface::connect when used generically?
``` cpp
WiFiInterface wifi;
wifi.set_network(ip, netmask, gw);

void do_thin(NetworkInterface *iface) {
    iface->connect();
    // do stuff
    iface->disconnect();
}
```